manifests/01_deployment.yaml

```diff
@@ -167,6 +169,9 @@ spec:
             cpu: 10m
             memory: 150Mi
         terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - name: cco-trusted-ca
+          mountPath: /etc/pki/ca-trust/extracted/pem/
```

Comment:
Does this behave ok when there's no proxy in play?

Reply:
supposedly. copied this from https://github.com/operator-framework/operator-marketplace/pull/245/files#diff-5e5f804a8ee2fb8c81711b728eb5a21dR76

Reply:
also why i pushed this PR. to see how CI handle this